invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/components/Header/Header.js

```diff
@@ -23,48 +25,46 @@ class Header extends Component {
         <Grid verticalAlign="middle">
           <Grid columns={2}>
             <Grid.Column width={4} textAlign="center">
-              <h1
-                className="logo"
-                onClick={() => this.goTo(BackOfficeRoutes.home)}
-              >
+              <h1 className="logo" onClick={goTo(BackOfficeRoutes.home)}>
```

Comment:
I noticed the introduction of the `goTo` but I am not sure for its necessity, cant we just do, 
`() => history.push(BackOfficeRoutes.home)`?

Reply:
It's introduced because is not recommendable in general to use anonymous functions on handlers and also I have created an issue so we can have only one place that we use to navigate :)